packages/perspective-jupyterlab/src/ts/view.ts

```diff
@@ -305,7 +314,12 @@ export class PerspectiveView extends DOMWidgetView {
                 } else {
                     // Just load the view into the widget, everything else
                     // is handled.
-                    this.pWidget.load(kernel_view, table_options);
+
+                    // TODO this should lilely be removed
+                    kernel_view.to_arrow().then((arrow: ArrayBuffer) => {
```

Comment:
This breaks - there is no `on_update` callback so when the table on the kernel updates, it will not reflect in the widget.